extensions/bitwarden/src/search.tsx

```diff
@@ -229,14 +229,13 @@ function PasswordActions(props: { password: string }) {
 
 function UsernameActions(props: { username: string }) {
```

Comment:
Please use singular here, you are only returning a single action

Reply:
I believe this is now complete?

Reply:
There might be an issue on my end, but I still see `UsernameActions` 😅 

Looks good otherwise ! Please add yourself to the list of contributors !

Reply:
My apologies folks, I misunderstood and thought you meant re. the props passthrough, but I have reverted that and renamed `UsernameActions` to `UsernameAction`, as well as added myself to the package.json. Thanks :)